packages/main/src/Link.js

```diff
@@ -116,26 +116,28 @@ const metadata = {
 		},
 
 		/**
-		 * Defines the aria-haspopup value of the component.
-		 *
-		 * @type String
-		 * @defaultvalue undefined
+		 * Defines the accessibility role of the component.
+		 * @defaultvalue ""
```

Comment:
Both recommendations make sense 
 - marking it with defaultValue: "link" makes it more clear in the docs,
 - list of supported values is also good to have
 
accessibleRole is an existing property, the github view is displaying it as addition, but it's not. so I don't see it like we immediately should address it in this change, as here it's more about the aria-expanded, aria-haspopip attrs

Reply:
Isn't it previously just role?  @unazko you should also modify role setting in the hbs. effectiveAccRole method should be removed and you should pass accessibleRole to the role attribute in the hbs. What is more, you should add tests and info in the commit for this. Maybe it is better to make those changes in a separate PR, since this one is for the accessibilityAttributes

Reply:
@unazko ignore my last comment, I though that this is a new property, but we should make it public.